src/clients/java/src/main/java/com/tigerbeetle/UInt128.java

```diff
@@ -172,5 +174,60 @@ public static byte[] asBytes(final BigInteger value) {
         return asBytes(bigintLsb.longValueExact(), bigintMsb.longValueExact());
     }
 
+    private static long idLastTimestamp = 0L;
+    private static final byte[] idLastRandom = new byte[80];
+    private static final SecureRandom idSecureRandom = new SecureRandom();
 
+    /**
+     * Generates a Universally Unique Sortable Identifier as 16 bytes of a 128-bit value.
+     *
+     * The ID() function is thread-safe, the bytes returned are formatted in little endian, and the
```

Comment:
I think it's worth mentioning that this ID is based on the UILD spec, except it is little-endian, so it is binary sortable and not lexicographically sortable by the string representation. In fact, our version has no concept of string representation.

```suggestion
     * The ID() function is thread-safe, the bytes returned are stored in little endian, and the
```